src/irc/__init__.py

```diff
@@ -183,3 +188,8 @@ def _new_message(data, value, numerology_func=number_to_numerology):
         data += " " + app
 
     return data
+
+
+def chunks(l, n):
+    n = max(1, n)
+    return (l[i : i + n] for i in range(0, len(l), n))
```

Comment:
This is longer but might be easier to read

```suggestion
    for i in range(0, len(message), n):
        yield message[i: i + n]
```